ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/JsonSchemaTranslator.java

```diff
@@ -15,19 +15,28 @@
 
 package io.confluent.ksql.serde.json;
 
+import com.google.common.collect.ImmutableMap;
 import io.confluent.connect.json.JsonSchemaData;
+import io.confluent.connect.json.JsonSchemaDataConfig;
 import io.confluent.kafka.schemaregistry.ParsedSchema;
 import io.confluent.kafka.schemaregistry.json.JsonSchema;
 import io.confluent.ksql.serde.connect.ConnectSchemaTranslator;
 import java.util.Map;
 import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.json.DecimalFormat;
+import org.apache.kafka.connect.json.JsonConverterConfig;
 
 /**
  * Translates between Connect and JSON Schema Registry schema types.
  */
-class JsonSchemaTranslator implements ConnectSchemaTranslator {
-
-  private final JsonSchemaData jsonData = new JsonSchemaData();
+public class JsonSchemaTranslator implements ConnectSchemaTranslator {
```

Comment:
Is there any discussion we need to have about making this class public?

Reply:
I need this class for the QTT tests. Proto and Avro schema translators are also public for QTT.